src/components/MessageList/hooks/MessageList/useMessageListElements.tsx

```diff
@@ -20,6 +14,7 @@ import type { MessageProps } from '../../../Message/types';
 import type { StreamMessage } from '../../../../context/ChannelStateContext';
 
 import type { DefaultStreamChatGenerics } from '../../../../types/types';
+import { MessageRenderer } from '../../renderMessages';
 
 type MessagePropsToOmit =
```

Comment:
I propose to get rid of the duplicate `MessagePropsToOmit` type so that we do not have to maintain it in multiple places.

Reply:
Ooops, that's actually a mistake - I'll get rid of the duplicate `MessagePropsToOmit`. Thanks for pointing it out!

Reply:
Fixed in https://github.com/GetStream/stream-chat-react/pull/2243/commits/75cd10bbc4bd86025636d519943b4ff91253583a.